# SystemInfo2.msg
std_msgs/Header header
uint64 ecu_hw_serial_sumber # ECU hardware serial number
uint32 eiu_sw_version # ECU software version